packages/bot-examples/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "@towns-protocol/bot-examples",
+    "version": "0.0.167",
+    "packageManager": "yarn@3.8.0",
+    "type": "module",
+    "scripts": {
+        "dev:survey": "NODE_TLS_REJECT_UNAUTHORIZED=0 tsx --env-file=.env src/survey-bot.ts",
+        "dev:thread-ai": "NODE_TLS_REJECT_UNAUTHORIZED=0 tsx --env-file=.env src/thread-ai-bot.ts",
```

Comment:
I'll break down those two examples into separate packages, so we can later use them as templates, like `create-towns-bot --template thread-ai`, copying the thread-ai bot package